nodes/test/memory_persistent/test_node.py

```
@@ -0,0 +1,1034 @@
+
+    Yields the mock entries dict so tests can inspect individual mocks.
+    Cleanup is automatic via monkeypatch.
+    """
+    entries = _build_engine_mock_entries()
+    for name, mod in entries.items():
+        monkeypatch.setitem(sys.modules, name, mod)
+    # Clear cached rocketlib modules so they re-import with the mocks
+    for key in list(sys.modules.keys()):
+        if key.startswith('rocketlib'):
+            monkeypatch.delitem(sys.modules, key, raising=False)
+    yield entries
```

Ruff (0.15.9)

[warning] No teardown in fixture `engine_mocks`, use `return` instead of `yield`

Replace `yield` with `return`

(PT022)